### PR TITLE
Add OVN to the E2E test matrix

### PR DESCRIPTION
**Pull request by skitt**

### CODE REVIEW

**Review by maayanf24 — APPROVED** [review]

**Comment by tpantelis** [comment]
Bumping **submariner-operator** version to pull in the RBAC changes also pulls in the embedded yaml changes so we need https://github.com/submariner-io/subctl/pull/883 to fix the build errors. I just included this commit with https://github.com/submariner-io/subctl/pull/883.

File: .github/workflows/e2e-full.yml [hunk 1]
```diff
@@ -18,14 +18,15 @@ jobs:
       matrix:
         globalnet: ['', 'globalnet']
         lighthouse: ['', 'lighthouse']
+        ovn: ['', 'ovn']
```
Comment (sridhargaddam) [inline .github/workflows/e2e-full.yml:21]: This looks fine, but as it doubles the number of jobs, can we add two additional jobs using the [include](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) options and validate the following additional jobs?
1. GN + LH + OVN
2. LH + OVN

Comment (tpantelis) [reply .github/workflows/e2e-full.yml:21]: Note that this commit is now part of https://github.com/submariner-io/subctl/pull/883. This PR adds 4 more jobs, so you're saying you want to exclude these?

1. OVN
2. OVN + GN

**Comment by tpantelis** [comment]
Included with https://github.com/submariner-io/subctl/pull/883

**Pull request closed without merging**